src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Cohort/DetailsViewModelMapper.cs

```diff
@@ -113,7 +113,9 @@ private async Task<IReadOnlyCollection<DetailsViewCourseGroupingModel>> GroupCou
                             OriginalStartDate = a.OriginalStartDate,
                             HasOverlappingEmail = emailOverlaps.Any(x=>x.Id == a.Id),
                             ULN = a.Uln,
-                            IsComplete = IsDraftApprenticeshipComplete(a, cohortResponse)
+                            IsComplete = IsDraftApprenticeshipComplete(a, cohortResponse),
```

Comment:
This function will need updating in the future, but I think we can get away with it for now